en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -57,15 +70,13 @@ The actions you can perform are as follows:
 
     ![View latency details](../assets/img/observability/latency-details.png)
     
-- **View details at a higher granularity:**
+- **View details for a custom time range:**
    
     If you want to drill down to view details at a higher granularity, drag the pointer over a time range to view the graph for the selected period.
     
-- **Check the status code of a request that has caused a latency:**
+- **View details per request:**
 
-    To do this, click the relevant latency value. In the following figure, the status code is `500`, which means the request resulted in an error state:
-    
-    ![Check error status code](../assets/img/observability/check-error-status-code.png){.cInlineImage-full}
+    Similar to the throughput graph, latency graph allows you to view details for each individual request that the service received during a selected custom time range. For details, see [Analyze the throughput graph](#analyze-the-throughput-graph) - **View details per request** bullet point.
```

Comment:
Should we remove "Similar to the throughput graph," and just start with "The latency graph allows you. . . . ".

Reply:
Done